<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wigle-Extract : 1.0 : LGPLv3 : https://github.com/EchterAlsFake/wigle-extract</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_csv">
       <property name="styleSheet">
        <string notr="true">color: white</string>
       </property>
       <property name="text">
        <string>CSV File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineedit_csv">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: rgb(107, 0, 255)
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_scan">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #5468ff;
	border-width: 2px;
	border-color: rgb(98, 255, 182);
	border-style: double;
    border-radius: 6px;
    color: #fff;
    font-family: &quot;JetBrains Mono&quot;, monospace;
    font-size: 15px;
    padding: 0px 16px 0px 16px;
    height: 24px;
}

QPushButton:hover {
    background-color: #3c4fe0;
	border-color: rgb(179, 0, 255)
}

QPushButton:pressed {
    background-color: #3c4fe0;
}
</string>
       </property>
       <property name="text">
        <string>Scan</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QTableWidget" name="table_widget_wpa">
        <property name="styleSheet">
         <string notr="true">background-color: gray;
color: white;</string>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Mac</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Lat</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Lon</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_tp_link">
        <property name="styleSheet">
         <string notr="true">color: white</string>
        </property>
        <property name="text">
         <string>TP-Link</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTableWidget" name="table_widget_wep">
        <property name="styleSheet">
         <string notr="true">background-color: gray;
color: white;</string>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>MAC</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Lat</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Lon</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_o2">
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>O2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTableWidget" name="table_widget_tp_link">
        <property name="styleSheet">
         <string notr="true">background-color: gray;
color: white;</string>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>MAC</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Lat</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Lon</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTableWidget" name="table_widget_o2">
        <property name="styleSheet">
         <string notr="true">background-color: gray;
color: white;
</string>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>MAC</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Lat</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Lon</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_wpa">
        <property name="styleSheet">
         <string notr="true">color: white</string>
        </property>
        <property name="text">
         <string>WPA</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_wep">
        <property name="styleSheet">
         <string notr="true">color: white</string>
        </property>
        <property name="text">
         <string>WEP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_wep_total">
        <property name="styleSheet">
         <string notr="true">color: white</string>
        </property>
        <property name="text">
         <string>WEP:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineedit_wep_total">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: rgb(107, 0, 255)
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_wpa_total">
        <property name="styleSheet">
         <string notr="true">color: white</string>
        </property>
        <property name="text">
         <string>WPA:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineedit_wpa_total">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: rgb(107, 0, 255)
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_o2_total">
        <property name="styleSheet">
         <string notr="true">color: white</string>
        </property>
        <property name="text">
         <string>O2:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineedit_o2_total">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: rgb(107, 0, 255)
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_tp_link_total">
        <property name="styleSheet">
         <string notr="true">color: white</string>
        </property>
        <property name="text">
         <string>TP-Link:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineedit_tp_link_total">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: rgb(107, 0, 255)
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_total">
        <property name="styleSheet">
         <string notr="true">color: white</string>
        </property>
        <property name="text">
         <string>Total:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineedit_total">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: rgb(107, 0, 255)
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
